Hero Section Revamp

Ellahinator opened the pull request:
- https://github.com/trypear/pear-landing-page/issues/59

Figma:
![image](https://github.com/sea-lene/pear-landing-page/assets/87584757/4fb63462-0809-4a9d-ab82-9d3c97782c36)

Before: 
![image](https://github.com/sea-lene/pear-landing-page/assets/87584757/b27e379c-b9c9-4ed6-a486-918175605473)

Implementation: 
![image](https://github.com/sea-lene/pear-landing-page/assets/87584757/e3a4c279-3c91-4833-be36-39b8cf364f40)


Notes:
- ~~I had trouble adding the logo to the left of "PearAI:" and would like some help.~~
  - @somwrks added the logo to the hero https://github.com/Ellahinator/pear-landing-page/pull/6

- Also am unsure what the `hover` behavior should be for the `More details` button.

MohammedADev commented on components/hero.tsx:
```diff
@@ -33,34 +36,63 @@ export default function Hero() {
+          <div className="mx-auto flex max-w-3xl flex-col items-center pb-12 text-center md:pb-16">
+            <div className="flex flex-col space-x-2 text-3xl font-semibold md:items-center md:text-4xl">
+              <div className="flex flex-row items-center space-x-2">
+                <PearHeroLogo width="26" className="hidden sm:block" />
```
It's a good practice to provide alternative text (alt attribute) for the <PearHeroLogo> component just in case the image fails to load or for screen reader users

MohammedADev replied:
that would be for both PearHeroLogo components, and the same alt text can be used for both

Ellahinator replied:
Done!

MohammedADev commented on components/hero.tsx:
```diff
@@ -33,34 +36,63 @@ export default function Hero() {
+              <span className="sm:hidden">AI-Powered Code Editor</span>
+              <span className="hidden sm:block">Code Editor</span>
+              {/*
+            </h1> */}
```
remove unnecessary comment

Ellahinator replied:
Done!

MohammedADev commented on components/hero.tsx:
```diff
@@ -33,34 +36,63 @@ export default function Hero() {
+                data-aos-delay="200"
+              >
+                Afraid of switching editors? No need, Pear is a fork of VSCode
+                and retains all its features, you'll feel right at home 😏
```
replace `you'll` with `you&apos;ll` otherwise Eslint will throw error

Ellahinator replied:
Done!

ICEPrey reviewed (changes requested):
very minor suggestions to improve code quality

ICEPrey reviewed (approved):
Shouldn't merge before add hover effect

Fryingpannn commented:
## Great PR! Just 2 things to change

1. Fix mobile responsiveness :) the "logo+pearai" should be its own line
<img width="428" alt="image" src="https://github.com/sea-lene/pear-landing-page/assets/59063950/0f489a78-1ea5-4e67-800f-1004a35a0d16">

2. For hover effect, accentuates the color more sharply/darker + slight shadow should do the trick!
![image](https://github.com/sea-lene/pear-landing-page/assets/59063950/7fc6bb33-6ea6-4277-bb27-a7444640dba8)
![image](https://github.com/sea-lene/pear-landing-page/assets/59063950/37df106c-186c-437e-99a5-8dd07161de39)

Good to go after these!

Ellahinator commented:
1. Fixed responsiveness for all sizes
2.  Added subtle hover effect on "More Details" button
    - darker border and darker text + shadow. also added shadow on "Join Waitlist" button
 
    ![image](https://github.com/sea-lene/pear-landing-page/assets/87584757/daf9f085-efa0-4267-9b7b-ba0587c1da7c)

    ![image](https://github.com/sea-lene/pear-landing-page/assets/87584757/5c1f372f-0b3b-4777-aecb-798f892f3ca4)
